### PR TITLE
Several usability tweaks and changing the namespace default

### PR DESCRIPTION
**Pull request by Gordonby**

### CODE REVIEW

File: local-process-quickstart.sh [hunk 1]
```diff
@@ -48,18 +60,18 @@ cleanupFunction()
...
    if [ "$RESPONSE" == "y" ] || [ "$RESPONSE" == "Y" ]; then
-      kubectl delete namespace dev
+      kubectl delete namespace $BIKENS
    fi
    rm -rf $HELMDIR
    exit 0
 }
 
-while getopts "g:n:r:cd" opt; do
+while getopts "g:n:rcd" opt; do
```
Comment (stepro) [inline local-process-quickstart.sh:74]: Was this an intentional change? If the `r` option needs a value, it needs the `:`

File: local-process-quickstart.sh [hunk 2]
```diff
@@ -70,8 +82,17 @@ while getopts "g:n:r:cd" opt; do
    esac
 done
 
+
+if [ -z "$BIKENS" ]; then
+   echo "Defaulting Kubernetes Namespace to: bikeapp"
+   BIKENS="bikeapp"
+fi
+BIKENS=${BIKENS,,}
```
Comment (stepro) [inline local-process-quickstart.sh:90]: What does this do? On my Mac, it gives an error `bad substitution`. What were you trying to do?

File: local-process-quickstart.sh [hunk 3]
```diff
@@ -70,8 +82,17 @@ while getopts "g:n:r:cd" opt; do
    esac
 done
 
+
+if [ -z "$BIKENS" ]; then
```
Comment (stepro) [inline local-process-quickstart.sh:86]: Is there not an option for this? Do you just need to know to set `BIKENS` as an environment variable before calling the script?

File: local-process-quickstart.sh (hunk 1 above)
Comment (Gordonby) [reply local-process-quickstart.sh:74]: r defaults to PWD, so it didn't feel necessary to have it there.

File: local-process-quickstart.sh (hunk 2 above)
Comment (Gordonby) [reply local-process-quickstart.sh:90]: I was allowing the namespace to be passed as an option - so was _making the namespace lowercase_.  It's only supported in newer bash versions, i didn't check Mac - so lets bin line 90

File: local-process-quickstart.sh (hunk 3 above)
Comment (Gordonby) [reply local-process-quickstart.sh:86]: I did put an option in for it, but the behaviour was unusual.  I postponed continuing with the option for this PR.  Lets talk this one over.

File: local-process-quickstart.sh (hunk 1 above)
Comment (stepro) [reply local-process-quickstart.sh:74]: Isn't the intended usage like `-r ../some/directory/that/isnt/pwd`? The getopts string needs to be `r:` for that to work. Or else it should be removed as an option.

File: local-process-quickstart.sh (hunk 2 above)
Comment (stepro) [reply local-process-quickstart.sh]: Ah. The cross-platform way that will work for that is `echo $BIKENS | tr [:upper:] [:lower:]`.

**Pull request merged**